memote/support/consistency.py

```diff
@@ -548,6 +548,36 @@ def find_metabolites_consumed_with_closed_bounds(model):
     return mets_consumed
 
 
+def find_metabolite_production_feasibility(model):
+    """
+    Return number of metabolites produced in complete medium.
```

Comment:
Wouldn't it be more interesting to know the metabolites that cannot be produced in complete medium? This has a large overlap with blocked reactions, deadend- and orphan metabolties, and isolated metabolites, of course.

Reply:
Actually, that is what is displayed on all the test, in the form of:
`ann["data"] = set(model.metabolites) - find_metabolite_production_feasibility(model)`

Reply:
I can change the function so it returns `infeasible_mets` however

Reply:
Whatever works better for you. Just seemed more straight forward this way around.